libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotMessageHandler.cs

```diff
@@ -27,7 +26,7 @@ public class BotMessageHandler : BotMessageHandlerBase
                     request.Headers["Authorization"],
                     activity,
                     botCallbackHandler,
-                    cancellationToken);
+                    cancellationToken).ConfigureAwait(false);
```

Comment:
So technically you want to come back to the ASP.NET context here. I would not suggest using `ConfigureAwait` in any ASP.NET component ever really because, like WinForms, they are purposefully working in a specific `SynchronizationContext`... even though it's not tied to a specific thread pool per se.

Reply:
humm ok so I'll take it out of the integration projects - there are only a few occurrences

Reply:
Good point.